packages/cloud-cognitive/src/components/CreateSidePanel/CreateSidePanel.stories.js

```diff
@@ -243,6 +246,117 @@ const TemplateWithFormValidation = ({ ...args }) => {
   );
 };
 
+const TemplateWithMultipleForms = ({ ...args }) => {
+  const [open, setOpen] = useState(false);
+  const [textInput, setTextInput] = useState('');
+  const [invalid, setInvalid] = useState(false);
+  return (
+    <>
+      {renderUIShellHeader()}
+      <Grid id="cloud-and-cognitive-page-content">
+        <Row>
+          <Column>
+            <Button onClick={() => setOpen(!open)}>
+              {open ? 'Close side panel' : 'Open side panel'}
+            </Button>
+          </Column>
+        </Row>
+      </Grid>
+      <CreateSidePanel
+        {...args}
+        slideIn={true}
```

Comment:
You can just add `slideIn` instead of `slideIn={true}` (don't need to set it to `true`). By adding it, it's implicitly true already.

Reply:
`slideIn` does not need to be added to a prop as `CreateSidePanel` because it's already added in `SidePanel` in `CreateSidePanel.js` (line 83) so I'm going to remove them altogether. Good catch 👍